### PR TITLE
Vulnerability disclamer and email.

### PR DESCRIPTION
**Pull request by nastasi-oq**
Add disclaimer and email contact to ask questions at the bottom of physical vulnerability curves view page .

### CODE REVIEW

**Comment by nastasi-oq** [comment]
Try to open the email link.
foto

File: openquakeplatform/openquakeplatform/vulnerability/templates/vulnerability/show_general_information.html [hunk 1]
```diff
@@ -134,6 +145,12 @@
 {% endblock messages %}
         <div id="mainApp" class="main_app">
         </div>
+        <div>
+          <div id="geninfo_footer">
+            <span>Disclaimer: some of the functions/curves have been digitized. Therefore their values may be different to those present in the original publication.</span><br>
+            For more information or comments about a curve/function contact: <a href="mailto:{{ user_mailaddr|urlencode }}?cc={{ vuln_mailaddr|urlencode }}&subject={{ vuln_mailsubj|urlencode }}">{{ user_mailaddr }}</a>.<br>
```
Comment (daniviga) [inline openquakeplatform/openquakeplatform/vulnerability/templates/vulnerability/show_general_information.html]: Please convert `&` to `&amp;`

**Comment by daniviga** [comment]
Apart for the minor comment about HTML entities, LGTM. I tested the `mailto:` feature on my Mac VM and it worked.

File: openquakeplatform/openquakeplatform/vulnerability/views.py [hunk 2]
```diff
@@ -748,7 +748,12 @@ def show_general_information(request, gen_info_id, action=None):
              edit_enabled=edit_enabled,
              curve_in_cart=curve_in_cart,
              cart_curves=cart_curves,
-             slug_name=slug_name),
+             slug_name=slug_name,
+             user_mailaddr="%s %s <%s>" % (
+                    gi.owner.first_name, gi.owner.last_name,
+                    gi.owner.email),
+             vuln_mailaddr="GEM Vulnerability Team <vulnerability@globalquakemodel.org>",
```
Comment (daniviga) [inline openquakeplatform/openquakeplatform/vulnerability/views.py]: Please be careful: this address does not exist (yet). Please check with @CatalinaYepes

**Comment by daniviga** [comment]
Apart for the minor comment about HTML entities and the address that does not exist, LGTM. I tested the mailto: feature on my Mac VM and it worked.

**Pull request merged**